bundles/org.openhab.binding.orbitbhyve/src/main/resources/OH-INF/config/config.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="bridge-type:orbitbhyve:bridge">
+		<parameter name="email" type="text" required="true">
+			<label>Email parameter</label>
```

Comment:
You could specify `<context>email</context>` to get a free validation.

Reply:
added context